projects/packages/search/src/dashboard/components/button/style.scss

```diff
@@ -0,0 +1,206 @@
+@import "../../scss/calypso-colors";
+@import "../../scss/color-functions";
+@import "../../scss/functions/rem";
+@import "../../scss/typography";
```

Comment:
We'll probably want to move towards using the base-styles package instead.

Reply:
will try to use the package in the following PR.